pkg/sentry/platform/kvm/machine_arm64.go

```diff
@@ -107,7 +107,7 @@ func (m *machine) dropPageTables(pt *pagetables.PageTables) {
 
```

Comment:
Hi,
I have checked the code between x86 and arm64.
It seems that there is no difference between these 2 functions.
Can we move it into machine.go?

Thanks.

Reply:
Thanks. It is a good idea to move dropPageTables() to machine.go. Will refine the patch.